.github/workflows/ci.yml

```diff
@@ -77,6 +78,10 @@ jobs:
         - build: macos
           os: macos-latest
           rust: nightly
+        - build: macos-arm
+          os: macos-11
+          rust: nightly
+          target: aarch64-apple-darwin
```

Comment:
I've tried running the `ci` workflow without this `target` specified, so `cross` isn't used for building, and all the runs have passed so far. Not sure if that's helpful.
https://github.com/arcsi42/ripgrep/actions/runs/1725214380

Reply:
Without the target is it building a universal binary?

Reply:
Does cargo build universal binaries on Apple silicon? I'm not sure if it does yet, https://github.com/rust-lang/cargo/issues/8875.
But we could use `lipo` in a `macos-11` runner to merge both the `x86_64` and `arm64/aarch64` builds.
I have successful runs using `lipo`: https://github.com/arcsi42/ripgrep/actions/runs/1729830482, https://github.com/arcsi42/ripgrep/releases/tag/TEST-0.0.3